frontend/src/components/NodeChart.vue

```diff
@@ -8,6 +8,18 @@
           max-width="100%"
           min-width="500px"
         >
+        <!-- Add Reset Zoom Icon -->
+        <v-btn
+          class="zoom-button"
+          color="#f4f4f4"
+          outlined
+          @click="resetZoom()"
+          style="position: absolute; top: 80px; right: 520px; z-index: 10;"
+          >
+          <v-icon :icon="mdiMagnifyMinusOutline" class="me-2"></v-icon>
+          RESET ZOOM
+        </v-btn> 
```

Comment:
suggest make this just icon with text on tooltip

Reply:
Great suggestion! Done!